builder/googlecompute/step_create_instance.go

```diff
@@ -114,9 +117,10 @@ func (s *StepCreateInstance) Run(ctx context.Context, state multistep.StateBag)
 	var errCh <-chan error
 	var metadata map[string]string
 	metadata, errs := c.createInstanceMetadata(sourceImage, string(c.Comm.SSHPublicKey))
-	if errs != nil && len(errs.Errors) > 0 {
+	if errs != nil {
 		state.Put("error", errs.Error())
 		ui.Error(errs.Error())
+		return multistep.ActionHalt
```

Comment:
@azr If there is any error, I think we should halt and return.

Reply:
I agree !